tools/old/tcpdrop.bt

```diff
@@ -35,7 +35,7 @@ BEGIN
   printf("%-8s %-8s %-16s %-21s %-21s %-8s\n", "TIME", "PID", "COMM", "SADDR:SPORT", "DADDR:DPORT", "STATE");
 
   // See https://github.com/torvalds/linux/blob/master/include/net/tcp_states.h
-  @tcp_states[1] = "ESTABLISHED";
+  @tcp_states[(uint64)1] = "ESTABLISHED";
```

Comment:
Reads a bit odd. How about using map declarations instead? Same for other scripts.

Reply:
you can't declare key/value types for map declarations.. yet

Reply:
ah, right